src/native/libs/System.Native/entrypoints.c

```diff
@@ -131,7 +131,7 @@ static const Entry s_sysNative[] =
     DllImportEntry(SystemNative_Malloc)
     DllImportEntry(SystemNative_Realloc)
     DllImportEntry(SystemNative_GetSpaceInfoForMountPoint)
-    DllImportEntry(SystemNative_GetFormatInfoForMountPoint)
+	DllImportEntry(SystemNative_GetFormatInfoForMountPoint)
```

Comment:
`git checkout HEAD~10 src/native` will undo whitespace changes in this directory.